applications/physics/ICF/pre_train_jag_wae.py

```diff
@@ -82,7 +82,7 @@ def construct_model():
     import lbann
 
     # Layer graph
-    input = lbann.Input(target_mode='N/A',data_set_per_model=True, name='inp_data')
```

Comment:
Why remove _data_set_per_model_ from here and other places?

Reply:
As a part of the design, this went away.  A data reader is global to a trainer and all models will share it.  We also deprecated the use case for the data set per model.